src/Command/SystemSetupCommand.php

```diff
@@ -53,6 +54,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             'SHOPWARE_CDN_STRATEGY_DEFAULT' => 'id',
             'BLUE_GREEN_DEPLOYMENT' => 1,
             'MAILER_URL' => 'smtp://localhost:25?encryption=&auth_mode=',
+            'COMPOSER_HOME' => $input->getOption('composer-home') ?: getenv('COMPOSER_HOME') ?: "{$this->projectDir}/var/cache/composer",
```

Comment:
Can we not use `$_SERVER['COMPOSER_HOME']` ?

Reply:
Yes we can :-)